app/poros/photo.rb

```diff
@@ -0,0 +1,7 @@
+class Photo
+  attr_reader :url
+  def initialize(details)
+    @url = details[:urls][:thumb]
+    @likes = details[:likes]
+  end
+end
```

Comment:
3 creating an instance with the bigggg json hash

4 pulling the thumbnail size photo link from the nested hash

5 put in there bc user story 4 deals with it